packages/tradershub/src/features/cfd/screens/DynamicLeverage/DynamicLeverageTableColumnHeader.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
+import { CaptionText, Text } from '@deriv/quill-design';
+
+type TDynamicLeverageTableColumnHeader = {
+    subtitle: string;
+    title: string;
+};
+
+export const DynamicLeverageTableColumnHeader: FC<TDynamicLeverageTableColumnHeader> = ({ subtitle, title }) => (
```

Comment:
Not much of a fan of having FC in these components will rather have TS infer jt
```suggestion
export const DynamicLeverageTableColumnHeader = ({ subtitle, title }:TDynamicLeverageTableColumnHeader) => (
```

Reply:
In my opinion it is unnecessary because it doesn't any value to the component and in addition we are just importing more stuff from `React`.

Reply:
I agree, updated.